crates/bevy_pbr/src/material.rs

```diff
@@ -386,8 +386,14 @@ pub fn queue_material_meshes<M: Material>(
                             MeshPipelineKey::from_primitive_topology(mesh.primitive_topology)
                                 | view_key;
                         let alpha_mode = material.properties.alpha_mode;
-                        if let AlphaMode::Blend = alpha_mode {
-                            mesh_key |= MeshPipelineKey::TRANSPARENT_MAIN_PASS;
+                        if let AlphaMode::Blend | AlphaMode::Premultiplied | AlphaMode::Add =
+                            alpha_mode
```

Comment:
Yeah, just double checked it, and it does work as expected 

[Playground example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=95bd23019392da1afb62dc61592a9098)

I didn't think much of it as I was making this change, since I just reused the existing `if let` statement, but looking back it does look kinda cleaner than `if alpha_mode == AlphaMode::Blend || alpha_mode == AlphaMode::Premultiplied || alpha_mode == AlphaMode::Add` 😄

Reply:
I agree it’s more concise. I think the mental hiccup for me is because the flags can be combined using the | or operator. But I don’t mind either way. This is fine.